python/plugins/processing/gui/ProcessingToolbox.py

```diff
@@ -149,19 +149,6 @@ def _filterItem(self, item, text):
                 show = (showChild or show) and item not in list(self.disabledProviderItems.values())
             item.setHidden(not show)
             return show
-        elif isinstance(item, (TreeAlgorithmItem, TreeActionItem)):
```

Comment:
isn't this still needed for TreeAlgorithmItem?

Reply:
Oups.